build.sbt

```diff
@@ -243,7 +243,7 @@ val main = root()
     riffRaffUploadManifestBucket := Some(
       System.getenv().getOrDefault("RIFF_RAFF_BUILD_BUCKET", "aws-frontend-teamcity"),
     ),
-    riffRaffManifestProjectName := s"dotcom:all",
+    riffRaffManifestProjectName := System.getenv().getOrDefault("RIFF_RAFF_PROJECT_NAME", "dotcom:all"),
```

Comment:
good q! I added this so that both TeamCity and GHA builds work simultaneously:

- the TeamCity build goes to `dotcom:all`
- we can override the project name for the GHA build, to `dotcom:frontend`

If we didn't have the option to set an override, both TC and GHA would build to the same project which could get confusing, especially as they have different build numbers.

Reply:
We should also remove this override once we merge the branch, and hardcode the new `dotcom:frontend` project name

Reply:
Good point - I added this mechanism when the upload of the compiled scala was still done by SBT's RiffRaff plugin. Now that all uploading is done in the GHA, I can remove this.